src/components/TrackNewExercise.tsx

```diff
@@ -61,6 +63,17 @@ export function TrackNewExercise(props: TrackNewExerciseProps): JSX.Element {
 
   const prepopulate = props.location.state
 
+  const formatMarkdown = (markdown: string | undefined): string | undefined => {
+    if (markdown) {
+      return prettier.format(markdown, {
+        parser: 'markdown',
```

Comment:
Fuk ye! Very cool.

Maybe we pass in the configuration we use in the repo as well? Just so that when prettier updates its "default", our code doesn't.

Reply:
Good idea! We only had two custom configuration settings. 

P.S. I had to also include the babel plugin, as that allows Prettier to format the JavaScript _within_ a Markdown file.

Reply:
Yeah the babel plugin makes sense 👍